docs/examples/snek5000-tgv/src/snek5000_tgv/tgv.usr.f

```diff
@@ -70,17 +70,42 @@ subroutine useric(ix,iy,iz,eg) ! set up initial conditions
       return
       end
 c-----------------------------------------------------------------------
-      subroutine userchk()
```

Comment:
I tend to prefer the version with empty parenthesis...

Could you also add `implicit none` for this function and other functions in this file?

Reply:
The problem is that `glsum` function doesn't have an implicit type.

Reply:
I guess you can add 
```
      ! functions
      real dnekclock_sync, glsum
```
See https://github.com/snek5000/snek5000-cbox/blob/main/src/snek5000_cbox/toolbox/chkpt_mstp.f#L668